wifiphisher/data/phishing-pages/wifi_connect/html/static/mac-network-manager.css

```diff
@@ -43,7 +43,7 @@
   color: white;
 }
 
-.mac-wifi .password input:focus + .glow {
+.mac-wifi .password  + .glow {
```

Comment:
What's the reason for this change?

Reply:
Because of this css (input:focus) you couldn't click(and then type) on the password field of mac enterprise networks. The cursor would stay permanently on the first field.